script.js

```diff
@@ -20,20 +21,27 @@ const moratoriumSheetId = "1AkYjbnLbWW83LTm6jcsRjg78hRVxWsSKQv1eSssDHSM";
 const renStikeSheetId = "1rCZfNXO3gbl5H3cKhGXKIv3samJ1KC4nLhCwwZqrHvU";
 
 // the URI that grabs the sheet text formatted as a CSV
-const moratoriumSheetURI = `https://docs.google.com/spreadsheets/d/${moratoriumSheetId}/export?format=csv&id=${moratoriumSheetId}`;
 const rentStrikeSheetURI = `https://docs.google.com/spreadsheets/d/${renStikeSheetId}/export?format=csv&id=${renStikeSheetId}`;
 
 // table in CARTO that syncs with the Google sheet data
 const cartoSheetSyncTable =
-  "emergency_tenant_protections_current_do_not_edit_me_sheet1";
+  "public.emergency_tenant_protections_scored";
 
 // the URIs for CARTO counties &s tates layers
 // joined to the moratoriums data
 // (all in AEMP CARTO acct)
 const cartoCountiesURI = createCountiesCartoURI();
 const cartoStatesURI = createStatesCartoURI();
+const cartoCitiesURI = createCitiesCartoURI();
 const cartoNationsURI = createNationsCartoURI();
 
+
+// colorScale comes from this ColorBrewer url:
+// https://colorbrewer2.org/#type=sequential&scheme=YlGn&n=7
+const colorNoData = "#939393";
+const fillColorScale = [colorNoData, "#d9f0a3", "#78c679", "#238443"];
+const strokeColorScale = [colorNoData, "#addd8e", "#41ab5d", "#005a32"];
```

Comment:
The stroke on states being different layers of green makes it hard to tell if states are coloured of not. The states in the middle, with no border, I thought they had the lowest rating. 
![border](https://user-images.githubusercontent.com/41007222/81470694-6cafee80-91ec-11ea-8801-cc2a5c6c6fa6.png)

I really think we should use the same border colour for all shapes, I just popped in an ugly #333 but it already is clearer. I think some lighter grey, but not too light, would help make it more clear

![border2](https://user-images.githubusercontent.com/41007222/81470755-cadcd180-91ec-11ea-820d-4c266d727a18.png)

Reply:
Oh I didn't realise the nation is on behind in the first pic. The point is that it's visually confusing, with the layer on, so having the same colour border would help with that problem.

Reply:
We have so many layers, and I think there's a lot of visual confusion due to: opacity, and not colouring regions with no data. it might be best to turn the layer select in to radio buttons instead of checkboxes.

Reply:
Another way we could use the existing border color scheme would be to do an `ORDER BY m.range` in the SQL queries for states, nations, and counties. This way the darkest boundary would always be drawn on top of the lightest, making it more apparent which is which.

<img width="1410" alt="image" src="https://user-images.githubusercontent.com/161748/81480752-5d2aa880-91e0-11ea-9ce8-4d8774d35d8b.png">

<img width="1338" alt="Screen Shot 2020-05-09 at 10 35 08 AM" src="https://user-images.githubusercontent.com/161748/81480859-35881000-91e1-11ea-9f2c-787054e264ac.png">

Reply:
we might need a `CASE` clause to assign a missing `range` value a 0 so that it's drawn on the bottom

Reply:
actually in the `SELECT` portion of the query, doing `COALESCE(range, 0) as range` should solve it.

Reply:
Eh, I just realized when layers are toggled on/off it messes up the ordering of the GeoJSON features, so doing this is starting to get more complex...